src/qnx/qinjector-glue.c

```diff
@@ -0,0 +1,738 @@
+#include "frida-core.h"
+
+#include <gio/gunixinputstream.h>
+#ifdef HAVE_I386
+# include <gum/arch-x86/gumx86writer.h>
+#endif
+#ifdef HAVE_ARM
+# include <gum/arch-arm/gumarmwriter.h>
+# include <gum/arch-arm/gumthumbwriter.h>
+#endif
+#include <gum/gum.h>
+#include <dlfcn.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <pthread.h>
+#include <stdio.h>
+#include <sys/mman.h>
+#include <sys/debug.h>
+#include <sys/procfs.h>
+#include <sys/stat.h>
+#include <sys/states.h>
+#include <sys/types.h>
+#ifdef HAVE_SYS_USER_H
+# include <sys/user.h>
+#endif
+#include <sys/wait.h>
```

Comment:
Includes should be sorted alphabetically. I suppose the I386 stuff isn't used btw?